contracts/PoolManager.sol

```diff
@@ -34,10 +35,10 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, ERC1155, IERC1155Rec
     using FeeLibrary for uint24;
 
     /// @inheritdoc IPoolManager
-    int24 public constant override MAX_TICK_SPACING = type(int16).max;
+    int24 public constant override MAX_TICK_SPACING = TickMath.MAX_TICK_SPACING;
```

Comment:
I know you didn't introduce this change but I think that the `override` keyword is unnecessary.

Reply:
Thanks! Removed both overrides.